authors/apps/articles/tests/test_tags.py

```diff
@@ -0,0 +1,87 @@
+from rest_framework import status
+import json
+
+from authors.apps.articles.tests.base_test import TestBaseCase
+
+
+class TagsTest(TestBaseCase):
+
+    def base_articles(self, message, response):
+        self.assertIn(message.encode(), response.content)
+
+    def authorized_post_request(self, url):
+        token = self.login_user()
+        response = self.client.post(
+            url, self.article, format='json',
+            HTTP_AUTHORIZATION='Token ' + token)
+        return response
+
+    def http_200_ok(self, response, message):
+        self.base_articles(message, response)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def http_201_created(self, response, message):
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.base_articles(message, response)
+
+    def http_403_forbidden(self, response):
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def create_articles(self):
+        token = self.login_user()
+        url = self.create_list_article_url
+        response = self.client.post(
+            url, self.article, format='json',
+            HTTP_AUTHORIZATION="Token " + token)
```

Comment:
Abstract away the authorization `HTTP_AUTHORIZATION="Token " + token)`,It's redundant in your tests .Check out my PR I have a function for this in base_test.py .You could copy paste it for now because my work isn't merged yet.

Reply:
Fixed